SKILL.tsv

```diff
@@ -4048,181 +4048,181 @@ SKILL_20150428_004047	{memo X}Summon Demon type boss{nl}Demon power +#{CaptionRa
+SKILL_20150714_004203	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Explosion Attack: #{CaptionRatio}#%{nl}Consumes 1 Hailmary Arrow
+SKILL_20150714_004204	$Attribute Attack: #{CaptionRatio}#%{nl}Duration: 10 seconds{nl}Number of hits: #{CaptionRatio2}#{nl}Consumes 1 Magic Arrow
+SKILL_20150714_004205	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Duration: 10 seconds{nl}Consumes 1 Divine Arrow
+SKILL_20150714_004206	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Can use while riding
 SKILL_20150714_004207	Fly circling around target area. Range defense of enemies within target area decreases.
-SKILL_20150714_004208	Range defense rate -#{CaptionRatio}# {nl}Duration 10 seconds
-SKILL_20150714_004209	Attack #{SkillAtkAdd}#{nl}Attribute Attack #{CaptionTime}#%{nl}HP recovery #{CaptionRatio}# ~ #{CaptionRatio2}#{nl}Magic Circle Duration 40 seconds
-SKILL_20150714_004210	Attack #{SkillAtkAdd}#{nl}Attribute Attack #{CaptionRatio2}#%{nl}Attack #{CaptionRatio}#times{nl}Magic Circle Duration #{CaptionTime}#seconds
-SKILL_20150714_004211	Attack #{SkillAtkAdd}#{nl}Attribute Attack #{CaptionRatio2}#%{nl}Attack #{CaptionRatio}#times{nl}Magic Circle Duration 30 seconds
-SKILL_20150714_004212	Attack #{SkillAtkAdd}#{nl}Attribute Attack #{CaptionRatio2}#%{nl}Defense #{CaptionRatio}# increases{nl}Target 10{nl}Duration #{CaptionTime}#seconds{nl}Use 1 Holy Water
-SKILL_20150714_004213	Attack #{SkillAtkAdd}#{nl}Attribute Attack #{CaptionRatio2}#%{nl}Duration #{CaptionTime}#seconds
-SKILL_20150714_004214	Attack #{SkillAtkAdd}#{nl}Attribute Attack #{CaptionTime}#%{nl}Attack bonus on 3rd attack #{CaptionRatio}#times ~ #{CaptionRatio2}#times
-SKILL_20150714_004215	Attack #{SkillAtkAdd}#{nl}Attribute Attack #{CaptionRatio}#%{nl}Summoned zombie required
+SKILL_20150714_004208	$Range Defense Ratio -#{CaptionRatio}# {nl}Duration: 10 seconds
+SKILL_20150714_004209	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionTime}#%{nl}HP Recovery #{CaptionRatio}# ~ #{CaptionRatio2}#{nl}Magic Circle Duration: 40 seconds
+SKILL_20150714_004210	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Number of hits: #{CaptionRatio}#{nl}Magic Circle Duration #{CaptionTime}# seconds
+SKILL_20150714_004211	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Number of hits: #{CaptionRatio}#{nl}Magic Circle Duration: 30 seconds
+SKILL_20150714_004212	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Defense +#{CaptionRatio}#{nl}Targets: 10{nl}Duration: #{CaptionTime}# seconds{nl}Consumes 1 Holy Water
+SKILL_20150714_004213	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio2}#%{nl}Duration: #{CaptionTime}# seconds
+SKILL_20150714_004214	$Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionTime}#%{nl}Bonus on 3rd Attack: x#{CaptionRatio}# ~ x#{CaptionRatio2}#
```

Comment:
I think "Bonus on 3rd Attack: #{CaptionRatio}# ~ #{CaptionRatio2}# times" would be better